src/shared/components/ScrollTable/ScrollTable.tsx

```diff
@@ -112,9 +113,15 @@ export const ScrollTableCore = <T extends object>({
                                     return (
                                         <ScrollTableCell
                                             key={id}
-                                            className={'align-' + stickyColumn.align}
+                                            className={
+                                                'align-' +
+                                                stickyColumn.align +
+                                                (hoveredRowIndex === id ? ' hover' : '')
+                                            }
```

Comment:
Few things with this one
1. Why not interpolation?
2. Why not `cx` (or `classnames`) lib.. We may still have it in deps. Manual classname construction looks nasty.
3. Why not a styled component?